model/src/main/proto/topic.proto

```diff
@@ -324,3 +324,41 @@ message RevisionCard {
   // translations.
   map<string, TranslationMapping> written_translation = 6;
 }
+
+// Top level proto used to store skill summaries.
+message SkillSummaryDatabase{
+  // Skill id to skill summary map.
+  map<string, SkillSummaryDomain> skill_summary_database = 1;
+}
+
+// A summary of a skill that can be trained or reviewed by the player which is saved in database.
```

Comment:
Nit: s/player/learner/

Reply:
I am actually not able to understand this comment correctly.
I have made this update:
`// A summary of a skill that can be trained or reviewed by the learner which is saved in database.`
Is this correct?